src/OrchardCore.Modules/OrchardCore.Facebook/Drivers/FacebookSettingsDisplayDriver.cs

```diff
@@ -87,19 +81,23 @@ public override async Task<IDisplayResult> UpdateAsync(FacebookSettings settings
                 var model = new FacebookSettingsViewModel();
                 await context.Updater.TryUpdateModelAsync(model, Prefix);
 
+                settings.AppId = model.AppId;
+                settings.FBInit = model.FBInit;
+                settings.SdkJs = model.SdkJs;
+                settings.Version = model.Version;
+
+                if (!string.IsNullOrWhiteSpace(model.FBInitParams))
+                {
+                    settings.FBInitParams = model.FBInitParams;
+                }
+
```

Comment:
Why move these out? There are some other `context.Updater.ModelState.IsValid`-related changes like this too.

Reply:
this are left overs after https://github.com/OrchardCMS/OrchardCore/pull/15488

We should always populate the model even if the validation failed

Reply:
Ah OK.